service/handlers_test.go

```diff
@@ -8,6 +8,7 @@ import (
 	"testing"
 
 	"github.com/dgrijalva/jwt-go"
+	"github.com/gorilla/context"
```

Comment:
See README: https://github.com/gorilla/context

Reply:
good point, we should use go's contexts for sure.

Reply:
just used the thing which was already used ;)

Reply:
yea I know, but we should actually remove gorillas context and start using the official go one. If it's not a problem for you, you can make that change too :)